docs/repl.md

```diff
@@ -17,7 +17,29 @@ This would open a Chrome browser that you can control with the REPL interface. M
 wdio repl chrome -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY
 ```
 
-You can apply any available options for your REPL session. (See `wdio --help` for options.)
+```sh
+wdio repl android
+OR
+wdio repl ios
+```
+
+This would open Chrome/Safari session on connected device/emulator/simulator. Make sure Appium running on port `4444` in order to initiate the session.
+
+```sh
+wdio repl './path/to/your_app.apk'
+```
+
+This would open App session on connected device/emulator/simulator. Make sure Appium running on port `4444` in order to initiate the session.
+
+Capabilities for iOS device can be passed with arguments:
+--ver     - platformVersion
+--device  - deviceName
+--udid    - udid (for real devices)
```

Comment:
Argument names should match the capability name. I think you could also add an alias to these flags via the yargs [positional](https://github.com/yargs/yargs/blob/master/docs/advanced.md#describing-positional-arguments) api.